synapse/handlers/_base.py

```diff
@@ -133,10 +133,9 @@ def handle_new_client_event(self, event, context, extra_destinations=[],
         for k, s in context.current_state.items():
             try:
                 if k[0] == EventTypes.Member:
-                    if s.content["membership"] == Membership.JOIN:
```

Comment:
Doesn't this make it impossible for servers to stop receiving events for a room after all their members in a room have left?